templates/success.html

```diff
@@ -3,23 +3,22 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>Registered to the meeting</title>
+  <title>Registration Successful</title>
 <link rel='stylesheet prefetch' href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/css/bootstrap.css'>
 
 </head>
 
 <body>
 
 <div class="container">
-  <h2>You are registered for the DESC meeting !</h2>
+  <h2>You are registered for the DESC meeting!</h2>
   <br>
   <br>
-  {{data.first_name}}, we are looking forward to seeing at our virtual meeting.
+  {{data.first_name}}, we are looking forward to seeing you at our virtual meeting.
   <br>
-  Please check your emails for confirmation of your payment.
+  Please check your emails for confirmation of your registration.
```

Comment:
Actually.... We don't send confirmation emails, do we? I used to happen automatically as a by-product of the payment, but we have no paymenet

Reply:
Ahh. Is it possible to send confirmations emails? This feature was requested by SOC. I wonder if the implementation will be tricky?

Reply:
hummmmmmmmmmmmmmmmmm It's doable of course, but not trivial :-/ most likely we would need to use an additional service

Reply:
Ok, let's leave that message for now, and we just need to find out how to send an email from the `registration_server.py`

Reply:
Then I think it's fine... they can check their names on Confluence anyway.